assets/docs/style.css

```diff
@@ -0,0 +1,136 @@
+.d-grid{
```

Comment:
Provided this was written by you, please add a GPLv3 and Tokiwa-header. Otherwise, where does this come from and what license does it have?